setup.py

```diff
@@ -101,23 +65,23 @@ def extensions():
     exts = [
         Extension(
             "mkl_random.mklrand",
-            [
-                os.path.join("mkl_random", "mklrand.pyx"),
-                os.path.join("mkl_random", "src", "mkl_distributions.cpp"),
-                os.path.join("mkl_random", "src", "randomkit.cpp"),
+            sources = [
+                join("mkl_random", "mklrand.pyx"),
+                join("mkl_random", "src", "mkl_distributions.cpp"),
+                join("mkl_random", "src", "randomkit.cpp"),
             ],
             depends = [
-                os.path.join("mkl_random", "src", "mkl_distributions.hpp"),
-                os.path.join("mkl_random", "src", "randomkit.h"),
-                os.path.join("mkl_random", "src", "numpy_multiiter_workaround.h")
+                join("mkl_random", "src", "mkl_distributions.hpp"),
+                join("mkl_random", "src", "randomkit.h"),
+                join("mkl_random", "src", "numpy_multiiter_workaround.h")
             ],
-            include_dirs = [os.path.join("mkl_random", "src"), np.get_include()] + mkl_include_dirs,
+            include_dirs = [join("mkl_random", "src"), np.get_include()] + mkl_include_dirs,
             libraries = libs,
             library_dirs = lib_dirs,
             extra_compile_args = eca + [
                 # "-ggdb", "-O0", "-Wall", "-Wextra",
```

Comment:
It seems legacy but unclear if that needs to be removed, since commented

Reply:
I meant it seems `extra_compile_args` is exactly the same as `eca`. Should we remove the commented code?

Reply:
yeah makes sense to